### ISSUE TITLE
Crashed while uninstalling Spotify app through AM

### ISSUE THREAD

**Issue by shuvashish76** [MuntashirAkon/AppManager#574]
- [x] I know what my device, OS and App Manager versions are
- [x] I know how to take logs
- [x] I know how to reproduce the issue which may not be specific to my device

**Describe the bug**
Crashed while uninstalling Spotify through AM. I guess it is something related to split apks or storage (my internal storage was almost full when I uninstalled Spotify app)

**To Reproduce**
Just go to App Info then click on Uninstall.

**Expected behavior**
It should uninstall Spotify app normally & it does uninstalled it. But AM crashed at the same time.

**Crash logs**

```
java.lang.IllegalArgumentException: Unknown component: ComponentInfo{com.spotify.music/com.facebook.CustomTabActivity}
    at android.os.Parcel.readException(Parcel.java:2017)
    at android.os.Parcel.readException(Parcel.java:1959)
    at android.content.pm.IPackageManager$Stub$Proxy.getComponentEnabledSetting(IPackageManager.java:4755)
    at android.app.ApplicationPackageManager.getComponentEnabledSetting(ApplicationPackageManager.java:2361)
    at io.github.muntashirakon.AppManager.details.AppDetailsFragment.isComponentDisabled(AppDetailsFragment.java:623)
    at io.github.muntashirakon.AppManager.details.AppDetailsFragment$AppDetailsRecyclerAdapter.getActivityView(AppDetailsFragment.java:916)
    at io.github.muntashirakon.AppManager.details.AppDetailsFragment$AppDetailsRecyclerAdapter.onBindViewHolder(AppDetailsFragment.java:896)
    at io.github.muntashirakon.AppManager.details.AppDetailsFragment$AppDetailsRecyclerAdapter.onBindViewHolder(AppDetailsFragment.java:644)
    at androidx.recyclerview.widget.RecyclerView$Adapter.onBindViewHolder(RecyclerView.java:7254)
    at androidx.recyclerview.widget.RecyclerView$Adapter.bindViewHolder(RecyclerView.java:7337)
    at androidx.recyclerview.widget.RecyclerView$Recycler.tryBindViewHolderByDeadline(RecyclerView.java:6194)
    at androidx.recyclerview.widget.RecyclerView$Recycler.tryGetViewHolderForPositionByDeadline(RecyclerView.java:6460)
    at androidx.recyclerview.widget.RecyclerView$Recycler.getViewForPosition(RecyclerView.java:6300)
    at androidx.recyclerview.widget.RecyclerView$Recycler.getViewForPosition(RecyclerView.java:6296)
    at androidx.recyclerview.widget.LinearLayoutManager$LayoutState.next(LinearLayoutManager.java:2330)
    at androidx.recyclerview.widget.LinearLayoutManager.layoutChunk(LinearLayoutManager.java:1631)
    at androidx.recyclerview.widget.LinearLayoutManager.fill(LinearLayoutManager.java:1591)
    at androidx.recyclerview.widget.LinearLayoutManager.onLayoutChildren(LinearLayoutManager.java:668)
    at androidx.recyclerview.widget.RecyclerView.dispatchLayoutStep2(RecyclerView.java:4309)
    at androidx.recyclerview.widget.RecyclerView.dispatchLayout(RecyclerView.java:4012)
    at androidx.recyclerview.widget.RecyclerView.onLayout(RecyclerView.java:4578)
    at android.view.View.layout(View.java:19663)
    at android.view.ViewGroup.layout(ViewGroup.java:6075)
    at android.widget.FrameLayout.layoutChildren(FrameLayout.java:323)
    at android.widget.FrameLayout.onLayout(FrameLayout.java:261)
    at android.view.View.layout(View.java:19663)
    at android.view.ViewGroup.layout(ViewGroup.java:6075)
    at androidx.appcompat.widget.LinearLayoutCompat.setChildFrame(LinearLayoutCompat.java:1655)
    at androidx.appcompat.widget.LinearLayoutCompat.layoutVertical(LinearLayoutCompat.java:1509)
    at androidx.appcompat.widget.LinearLayoutCompat.onLayout(LinearLayoutCompat.java:1417)
    at android.view.View.layout(View.java:19663)
    at android.view.ViewGroup.layout(ViewGroup.java:6075)
    at androidx.swiperefreshlayout.widget.SwipeRefreshLayout.onLayout(SwipeRefreshLayout.java:688)
    at android.view.View.layout(View.java:19663)
    at android.view.ViewGroup.layout(ViewGroup.java:6075)
    at androidx.viewpager.widget.ViewPager.onLayout(ViewPager.java:1775)
    at android.view.View.layout(View.java:19663)
    at android.view.ViewGroup.layout(ViewGroup.java:6075)
    at com.google.android.material.appbar.HeaderScrollingViewBehavior.layoutChild(HeaderScrollingViewBehavior.java:148)
    at com.google.android.material.appbar.ViewOffsetBehavior.onLayoutChild(ViewOffsetBehavior.java:43)
    at com.google.android.material.appbar.AppBarLayout$ScrollingViewBehavior.onLayoutChild(AppBarLayout.java:2003)
    at androidx.coordinatorlayout.widget.CoordinatorLayout.onLayout(CoordinatorLayout.java:918)
    at android.view.View.layout(View.java:19663)
    at android.view.ViewGroup.layout(ViewGroup.java:6075)
    at android.widget.FrameLayout.layoutChildren(FrameLayout.java:323)
    at android.widget.FrameLayout.onLayout(FrameLayout.java:261)
    at android.view.View.layout(View.java:19663)
    at android.view.ViewGroup.layout(ViewGroup.java:6075)
    at android.widget.LinearLayout.setChildFrame(LinearLayout.java:1791)
    at android.widget.LinearLayout.layoutVertical(LinearLayout.java:1635)
    at android.widget.LinearLayout.onLayout(LinearLayout.java:1544)
    at android.view.View.layout(View.java:19663)
    at android.view.ViewGroup.layout(ViewGroup.java:6075)
    at android.widget.FrameLayout.layoutChildren(FrameLayout.java:323)
    at android.widget.FrameLayout.onLayout(FrameLayout.java:261)
    at android.view.View.layout(View.java:19663)
    at android.view.ViewGroup.layout(ViewGroup.java:6075)
    at android.widget.LinearLayout.setChildFrame(LinearLayout.java:1791)
    at android.widget.LinearLayout.layoutVertical(LinearLayout.java:1635)
    at android.widget.LinearLayout.onLayout(LinearLayout.java:1544)
    at android.view.View.layout(View.java:19663)
    at android.view.ViewGroup.layout(ViewGroup.java:6075)
    at android.widget.FrameLayout.layoutChildren(FrameLayout.java:323)
    at android.widget.FrameLayout.onLayout(FrameLayout.java:261)
    at com.android.internal.policy.DecorView.onLayout(DecorView.java:761)
    at android.view.View.layout(View.java:19663)
    at android.view.ViewGroup.layout(ViewGroup.java:6075)
    at android.view.ViewRootImpl.performLayout(ViewRootImpl.java:2500)
    at android.view.ViewRootImpl.performTraversals(ViewRootImpl.java:2216)
    at android.view.ViewRootImpl.doTraversal(ViewRootImpl.java:1396)
    at android.view.ViewRootImpl$TraversalRunnable.run(ViewRootImpl.java:6773)
    at android.view.Choreographer$CallbackRecord.run(Choreographer.java:966)
    at android.view.Choreographer.doCallbacks(Choreographer.java:778)
    at android.view.Choreographer.doFrame(Choreographer.java:713)
    at android.view.Choreographer$FrameDisplayEventReceiver.run(Choreographer.java:952)
    at android.os.Handler.handleCallback(Handler.java:790)
    at android.os.Handler.dispatchMessage(Handler.java:99)
    at android.os.Looper.loop(Looper.java:164)
    at android.app.ActivityThread.main(ActivityThread.java:6518)
    at java.lang.reflect.Method.invoke(Native Method)
    at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:438)
    at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:807)
```

**Device Info**
App version: 2.6.3 (F-Droid version)
App version code: 392
Android build version: 8704X_S001025_190606_ROW
Android release version: 8.1.0
Android SDK version: 27
Android build ID: TB-8704X_S001025_190606_ROW
Device brand: Lenovo
Device manufacturer: LENOVO
Device name: TB-8704X
Device model: Lenovo TB-8704X
Device product name: TB-8704X
Device hardware name: qcom
ABIs: [arm64-v8a, armeabi-v7a, armeabi]
ABIs (32bit): [armeabi-v7a, armeabi]
ABIs (64bit): [arm64-v8a]
System language: en-IN
In-App Language: auto
Mode: auto

**Comment by MuntashirAkon** [comment]
Could be fixed in 8ac3de4b

**Comment by MuntashirAkon** [comment]
Closed due to lack of response. Feel free to open it again if you're still facing the issue.

**Comment by shuvashish76** [comment]
Last time I reported for v2.6.3 (392), now AM crashed with v2.6.5.1 (397)
Sorry fixed with which AM version?

<details>
  <summary>Crash logs</summary>
```
java.lang.IllegalArgumentException: Unknown component: ComponentInfo{com.amaze.fileutilities.debug/com.amaze.fileutilities.audio_player.AudioPlayerDialogActivity}
    at android.os.Parcel.readException(Parcel.java:2017)
    at android.os.Parcel.readException(Parcel.java:1959)
    at android.content.pm.IPackageManager$Stub$Proxy.getComponentEnabledSetting(IPackageManager.java:4755)
    at android.app.ApplicationPackageManager.getComponentEnabledSetting(ApplicationPackageManager.java:2361)
    at io.github.muntashirakon.AppManager.details.AppDetailsFragment.isComponentDisabled(AppDetailsFragment.java:627)
    at io.github.muntashirakon.AppManager.details.AppDetailsFragment$AppDetailsRecyclerAdapter.getActivityView(AppDetailsFragment.java:964)
    at io.github.muntashirakon.AppManager.details.AppDetailsFragment$AppDetailsRecyclerAdapter.onBindViewHolder(AppDetailsFragment.java:902)
    at io.github.muntashirakon.AppManager.details.AppDetailsFragment$AppDetailsRecyclerAdapter.onBindViewHolder(AppDetailsFragment.java:648)
    at androidx.recyclerview.widget.RecyclerView$Adapter.onBindViewHolder(RecyclerView.java:7254)
    at androidx.recyclerview.widget.RecyclerView$Adapter.bindViewHolder(RecyclerView.java:7337)
    at androidx.recyclerview.widget.RecyclerView$Recycler.tryBindViewHolderByDeadline(RecyclerView.java:6194)
    at androidx.recyclerview.widget.RecyclerView$Recycler.tryGetViewHolderForPositionByDeadline(RecyclerView.java:6460)
    at androidx.recyclerview.widget.RecyclerView$Recycler.getViewForPosition(RecyclerView.java:6300)
    at androidx.recyclerview.widget.RecyclerView$Recycler.getViewForPosition(RecyclerView.java:6296)
    at androidx.recyclerview.widget.LinearLayoutManager$LayoutState.next(LinearLayoutManager.java:2330)
    at androidx.recyclerview.widget.LinearLayoutManager.layoutChunk(LinearLayoutManager.java:1631)
    at androidx.recyclerview.widget.LinearLayoutManager.fill(LinearLayoutManager.java:1591)
    at androidx.recyclerview.widget.LinearLayoutManager.onLayoutChildren(LinearLayoutManager.java:668)
    at androidx.recyclerview.widget.RecyclerView.dispatchLayoutStep2(RecyclerView.java:4309)
    at androidx.recyclerview.widget.RecyclerView.dispatchLayout(RecyclerView.java:4012)
    at androidx.recyclerview.widget.RecyclerView.onLayout(RecyclerView.java:4578)
    at android.view.View.layout(View.java:19663)
    at android.view.ViewGroup.layout(ViewGroup.java:6075)
    at android.widget.FrameLayout.layoutChildren(FrameLayout.java:323)
    at android.widget.FrameLayout.onLayout(FrameLayout.java:261)
    at android.view.View.layout(View.java:19663)
    at android.view.ViewGroup.layout(ViewGroup.java:6075)
    at androidx.appcompat.widget.LinearLayoutCompat.setChildFrame(LinearLayoutCompat.java:1655)
    at androidx.appcompat.widget.LinearLayoutCompat.layoutVertical(LinearLayoutCompat.java:1509)
    at androidx.appcompat.widget.LinearLayoutCompat.onLayout(LinearLayoutCompat.java:1417)
    at android.view.View.layout(View.java:19663)
    at android.view.ViewGroup.layout(ViewGroup.java:6075)
    at androidx.swiperefreshlayout.widget.SwipeRefreshLayout.onLayout(SwipeRefreshLayout.java:688)
    at android.view.View.layout(View.java:19663)
    at android.view.ViewGroup.layout(ViewGroup.java:6075)
    at androidx.viewpager.widget.ViewPager.onLayout(ViewPager.java:1775)
    at android.view.View.layout(View.java:19663)
    at android.view.ViewGroup.layout(ViewGroup.java:6075)
    at com.google.android.material.appbar.HeaderScrollingViewBehavior.layoutChild(HeaderScrollingViewBehavior.java:148)
    at com.google.android.material.appbar.ViewOffsetBehavior.onLayoutChild(ViewOffsetBehavior.java:43)
    at com.google.android.material.appbar.AppBarLayout$ScrollingViewBehavior.onLayoutChild(AppBarLayout.java:2003)
    at androidx.coordinatorlayout.widget.CoordinatorLayout.onLayout(CoordinatorLayout.java:918)
    at android.view.View.layout(View.java:19663)
    at android.view.ViewGroup.layout(ViewGroup.java:6075)
    at android.widget.FrameLayout.layoutChildren(FrameLayout.java:323)
    at android.widget.FrameLayout.onLayout(FrameLayout.java:261)
    at android.view.View.layout(View.java:19663)
    at android.view.ViewGroup.layout(ViewGroup.java:6075)
    at android.widget.LinearLayout.setChildFrame(LinearLayout.java:1791)
    at android.widget.LinearLayout.layoutVertical(LinearLayout.java:1635)
    at android.widget.LinearLayout.onLayout(LinearLayout.java:1544)
    at android.view.View.layout(View.java:19663)
    at android.view.ViewGroup.layout(ViewGroup.java:6075)
    at android.widget.FrameLayout.layoutChildren(FrameLayout.java:323)
    at android.widget.FrameLayout.onLayout(FrameLayout.java:261)
    at android.view.View.layout(View.java:19663)
    at android.view.ViewGroup.layout(ViewGroup.java:6075)
    at android.widget.LinearLayout.setChildFrame(LinearLayout.java:1791)
    at android.widget.LinearLayout.layoutVertical(LinearLayout.java:1635)
    at android.widget.LinearLayout.onLayout(LinearLayout.java:1544)
    at android.view.View.layout(View.java:19663)
    at android.view.ViewGroup.layout(ViewGroup.java:6075)
    at android.widget.FrameLayout.layoutChildren(FrameLayout.java:323)
    at android.widget.FrameLayout.onLayout(FrameLayout.java:261)
    at com.android.internal.policy.DecorView.onLayout(DecorView.java:761)
    at android.view.View.layout(View.java:19663)
    at android.view.ViewGroup.layout(ViewGroup.java:6075)
    at android.view.ViewRootImpl.performLayout(ViewRootImpl.java:2500)
    at android.view.ViewRootImpl.performTraversals(ViewRootImpl.java:2216)
    at android.view.ViewRootImpl.doTraversal(ViewRootImpl.java:1396)
    at android.view.ViewRootImpl$TraversalRunnable.run(ViewRootImpl.java:6773)
    at android.view.Choreographer$CallbackRecord.run(Choreographer.java:966)
    at android.view.Choreographer.doCallbacks(Choreographer.java:778)
    at android.view.Choreographer.doFrame(Choreographer.java:713)
    at android.view.Choreographer$FrameDisplayEventReceiver.run(Choreographer.java:952)
    at android.os.Handler.handleCallback(Handler.java:790)
    at android.os.Handler.dispatchMessage(Handler.java:99)
    at android.os.Looper.loop(Looper.java:164)
    at android.app.ActivityThread.main(ActivityThread.java:6518)
    at java.lang.reflect.Method.invoke(Native Method)
    at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:438)
    at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:807)

Device Info:
App version: 2.6.5.1
App version code: 397
Android build version: 8704X_S001025_190606_ROW
Android release version: 8.1.0
Android SDK version: 27
Android build ID: TB-8704X_S001025_190606_ROW
Device brand: Lenovo
Device manufacturer: LENOVO
Device name: TB-8704X
Device model: Lenovo TB-8704X
Device product name: TB-8704X
Device hardware name: qcom
ABIs: [arm64-v8a, armeabi-v7a, armeabi]
ABIs (32bit): [armeabi-v7a, armeabi]
ABIs (64bit): [arm64-v8a]
System language: en-IN
In-App Language: auto
Mode: auto
```
<details

**Comment by MuntashirAkon** [comment]
Fixed in d065dc6f
